windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -104,6 +115,8 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 [id="wmco-2-0-0-known-issues"]
 == Known issues
 
+* When you create Windows pods with `RunAsUserName` set in its "SecurityContext" with a projected volume associated with these pods, the file ownership permissions for the projected entities are ignored, resulting in incorrectly configured ownership permissions.
```

Comment:
Feel free to ignore this comment, but this reads a bit odd to me (probably because I'm not familiar with it). Is "SecurityContext" a field `RunAsUserName` the value? If so, maybe:

When you create Windows pods with the `RunAsUserName` value set in the `SecurityContext`field with...

Reply:
@aravindhp Can you address this comment? I was a bit unsure and was using context clues. I believe SecurityContext is a permissions layer with `RunAsUserName` being an option.

Reply:
For context wecare talking about [Pod SecurityContext](https://docs.openshift.com/container-platform/4.7/rest_api/objects/index.html#podsecuritycontext-core-v1) here. Maybe we need to link to that?

Reply:
Excellent! I updated the sentence where we reference the SecurityContext with a link to that Pod SecurityContext documentation.